client/components/resource-details/ResourceInfo.tsx

```diff
@@ -144,29 +137,164 @@ const Reviews = ({ reviews }: ReviewsProps) => {
   return (
     <div>
       <h1>Reviews</h1>
-      {reviews.map((review) => {
-        return <ReviewEntry review={review} />
+      {reviews.map((review, index) => {
+        return <ReviewCard key={index} review={review} />
       })}
     </div>
   )
 }
 
 const Avatar = styled.img`
-  border-radius: 4px;
+  border-radius: 50%;
   display: block;
-  margin-top: 40px;
-  height: 100px;
-  width: 100px;
+  margin-left: 24px;
+  margin-right: 24px;
+  margin-top: 24px;
+  height: 80px;
+  width: 80px;
+`
+
+const RatingList = styled.div`
+  display: flex;
+  justify-content: flex-end;
+  flex-direction: column;
+  margin: 24px;
+  margin-left: auto;
+  align-self: flex-start;
+  @media ${smallerThan(width.mobileS)} {
+    margin-left: 24px;
+  }
+`
+
+const ReviewTopRow = styled.div`
+  display: flex;
+  @media ${smallerThan(width.laptop)} {
+    flex-wrap: wrap;
+  }
+`
+
+const ShowOnLaptop = styled.div`
+  @media ${smallerThan(width.laptop)} {
+    display: none;
+  }
 `
 
-const Rating = styled.p``
+const HideOnLaptop = styled.div`
```

Comment:
I really like this idea, we might even want to factor this into a util file under `styles` someday